pipelines/measurement/measurement_detector_table.yaml

```diff
@@ -0,0 +1,12 @@
+description: Compute per-detector metrics from sourceTable_visit catalogs
+tasks:
+  nsrcMeasDetectorTable:
+    class: lsst.faro.measurement.DetectorTableMeasurementTask
+    config:
+      connections.package: info
+      connections.metric: nsrcMeasDetectorTable
+      #connections.refDataset: 'gaia_dr2_20200414'
+      connections.refDataset: 'ps1_pv3_3pi_20170110'
```

Comment:
Rather than have a commented connection, if the goal here is to document which catalogs are available, I'd much rather they be listed in a comment rather than have the options be commented in valid code.  I.e. if it's there for purely informational purposes, it should be in a comment to make that clear.

Reply:
Made Gaia the default. Listing Gaia and PS1 as a separate comment now instead of commented options.